### PR TITLE
[BI-23] BE for login page

### PR DESCRIPTION
**Pull request by Lucian2**

### CODE REVIEW

File: src/views/Login.vue [hunk 1]
```diff
@@ -45,15 +50,33 @@
...
+     password:''
      },
     }
-  }
+  },
+  methods: {
+    async handleSubmit() {
+ try {
+      this.loading = true;
+       await authService.login(this.model);
```
Comment (Ceciliaa) [inline src/views/Login.vue:72]: When you receive the response, you should save the token

File: src/views/Login.vue [hunk 2]
```diff
@@ -45,15 +50,36 @@
 </template>
 
 <script>
+import BaseButton from '@/components/BaseButton.vue';
+import authService from '@/api/authService';
+import { get } from 'lodash';
```
Comment (Ceciliaa) [inline src/views/Login.vue:55]: Import just the get method
import get from 'loadash/get'

**Review by Ceciliaa — APPROVED** [review]

File: src/views/Login.vue (hunk 2 above)
Comment (LAlex14) [reply src/views/Login.vue:55]: + nu trebuie importat BaseButton, pentru ca e inclus in pluginul de globalComponents si se importa global

File: src/views/Login.vue [hunk 3]
```diff
@@ -45,15 +50,36 @@
...
-      password:''
+     email:'',
+     password:''
      },
     }
-  }
+  },
+  methods: {
+    async handleSubmit() {
+ try {
```
Comment (alexstefan-binarcode) [inline src/views/Login.vue:71]: Indent this a bit

File: src/views/Login.vue [hunk 4]
```diff
@@ -45,15 +50,36 @@
...
+      this.loading = true;
+        let {data} =  await authService.login(this.model)
+        const token = get(data, 'token', '')
+        authService.setToken(token)
+      } catch (error) {
+        console.log(error)
+      } finally {
+        this.loading = false;
+      }
+      this.$router.push('/');
```
Comment (alexstefan-binarcode) [inline src/views/Login.vue:81]: I would push the route only if the login is successful (after setting the token). Else, display a error message

**Review by alexstefan-binarcode — CHANGES_REQUESTED** [review]
Check the comments

**Review by alexstefan-binarcode — APPROVED** [review]
It looks ok

**Pull request merged**